edx/edx-ora2
# feat: Update Instructor dashboard Open Responses tab to be aware of ESG

**TL;DR -** 
The Instructor dashboard needs to be aware of the ESG waffle flag to provide either the old ORA experience or the new ESG experience at a course by course basis.

JIRA: [AU-52](https://openedx.atlassian.net/browse/AU-52)

**What changed?**

- ORA js fragment now receive `ENHANCED_STAFF_GRADER` from js context

**Developer Checklist**


**Testing Instructions**

- add `console.log(this.data.ENHANCED_STAFF_GRADER)` to [this line](https://github.com/edx/edx-ora2/blob/bd7483697c42515e2af3f0d68eb09a056d2ff9b4/openassessment/xblock/static/js/src/lms/oa_course_items_listing.js#L235)
- Create a test course
- Confirm or toggle the waffle flag `openresponseassessment.enhanced_staff_grader` on for the course. Assuming this is the flag from [AU-50](https://openedx.atlassian.net/browse/AU-50)
- Navigate to the Instructor dashboard -> Open Response
- The console should log `true`
- Toggle the waffle flag `openresponseassessment.enhanced_staff_grader` off for the course
- Navigate to the Instructor dashboard -> Open Response
- The console should log `false`

**Reviewer Checklist**

Collectively, these should be completed by reviewers of this PR:


FYI: @edx/masters-devs-gta
